CHANGELOG.md

```diff
@@ -49,6 +49,23 @@
 
 #### [nidcpower] Unreleased
 - Added
+  - API parity with NI-DCPower 2025 Q2.
+      - Properties added:
+        - `constant_power_compensation_frequency`
+        - `constant_power_current_limit`
+        - `constant_power_gain_bandwidth`
+        - `constant_power_level`
+        - `constant_power_level_range`
+        - `constant_power_pole_zero_ratio`
+        - `constant_resistance_compensation_frequency`
+        - `constant_resistance_current_limit`
+        - `constant_resistance_gain_bandwidth`
+        - `constant_resistance_level`
+        - `constant_resistance_level_range`
+        - `constant_resistance_pole_zero_ratio`
+        - `output_shorted`
+      - Enum values added:
+        - `CONSTANT_RESISTANCE` and `CONSTANT_POWER` added to enum `OutputFunction`
```

Comment:
All of this is under a bullet called "Added".
There is no need to keep saying "added" multiple times as long as it's under the Added bullet.

Reply:
I've removed the redundant "added"s from line 53 and 67 in the latest commit.